src/pfe/portal/modules/LoadRunner.js

```diff
@@ -582,7 +600,7 @@ module.exports = class LoadRunner {
       if (data.projectID === this.project.projectID) {
         log.info(`Load run on project ${this.project.projectID} cancelled`);
         this.heartbeat('cancelling');
-        this.cancelProfiling();
+        await this.cancelProfiling();
```

Comment:
@tobespc - I asked @ebuckle about the await here and he advised testing this as we changed codewind doesn't start/stop liberty during cancelling phase. Cancelling a liberty run seemed fine anbd awaiting is probably safer to make sure we've finished cancelling before we report that we've cancelled.